go-controller/pkg/ovn/admin_network_policy_test.go

```diff
@@ -0,0 +1,930 @@
+package ovn
```

Comment:
YES, I actually started doing this in a cleanup PR and it became a headache... will follow it up once we finish this PR
so I want to do a cleanup PR where we package the fakeOVN in such a way that it can be used from anywhere and things like `newPod` `newNamespace` should be reused instead of each controller suite creating its own set of functions..like APB has a newPod that is just a copy of newPod of newPod of the pods_test.go :/

However I still want to use the `oc` controller test logic to get that wholistic pods/namespaces add with anp/banp add together -> same problem lies with egressSVC, so my plan is to split default controller/base controller etc into their own packages -> else its hard to split fakeOVN into its own package and for other controllers to use the default controller invoker for testing things.. WDYT?